templates/docs/patterns/cta-section/index.md

```diff
@@ -0,0 +1,214 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: CTA Section | Patterns
+---
+
+{% from "docs/macros/patterns/wip-notice.jinja" import pattern_wip_notice %}
+
+{{- pattern_wip_notice() }}
+
+A CTA section is a prominent section typically used to quickly capture user's attention to a specific action as they scroll down the page.
+Depending on the design of call to action buttons or links, you can choose from a variety of layouts:
+
+- [Default full width](#default-full-width)
+- [Default full width cta html](#default-full-width-cta-html)
+- [Default 25-75](#default-25-75)
+- [Default 25-75 cta html](#default-25-75-cta-html)
+- [Block full width](#block-full-width)
+- [Block 25-75](#block-25-75)
+
+The CTA section pattern is composed of the following elements:
+
+| Element              | Description                                                   |
+| -------------------- | ------------------------------------------------------------- |
+| Rule                 | [Default rule](/docs/patterns/rule#default)                   |
+| Title                | Title text (to be placed in `h2` heading)                     |
+| Link                 | Link HTML or Text (using `h2` style)                          |
+| Description          | Description text (one or more paragraphs)                     |
+| Call to action block | [CTA block](/docs/patterns/cta-block) beneath the description |
+
+## Default full width
+
+You can use <code>default</code> variant with <code>100</code> layout to create a full width container with title and link stacked vertically.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/cta-section/default-full-width" class="js-example" data-lang="jinja">
+View example of the default full width cta section pattern
+</a></div>
+
+## Default full width links within text
+
+If you need to pass a text that can have multiple partial links within it, you can use do so as well.
```

Comment:
the "you can use do so as well" in this and the following sections is a bit wordy, I'd suggest rewording